wiki/Tournaments/NPC/2/en.md

```diff
@@ -0,0 +1,218 @@
+---
+
+tags:
+  - NPC
+  - NPC2
+---
+
+# Non-Professional Cup 2: Summer 2020
+
+![NPC2 logo](images/logo.png)
```

Comment:
This image is too big, please use JPEG 80%, or downsize the image to keep it under 1 MB

Reply:
The logo size has been shrinked